src/core/link-to-dfn.js

```diff
@@ -125,10 +127,17 @@ export async function run(conf, doc, cb) {
     }
   });
 
+  const additionaExternalLinks = [
+    ...document.querySelectorAll(
+      "a[data-cite]:not([data-cite='']):not([data-cite*='#'])"
+    ),
+  ].filter(el => {
```

Comment:
Looks like this is mapping? 

.map(el=> el.closest("[data-cite]")).filter(closest = > !closest || closest.dataset.cite !== "")

Reply:
No. Not a mapping here.
The present one takes into account links of type `<section data-cite="foo"><a>bar</a></section>` and `<section><a data-cite="foo">bar</a></section>`. 
What we need in `additionalLinks` is the `el` itself, and not `el.closest()`

Reply:
ok, cool.